oximeter/instruments/schema/http-service.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Silly question, but any reason this isn't minified, but `request-latency-histogram.json` is?

Reply:
Oh, that's probably because I made the pretty-printing after the file already existed, so the equality check passed. I'll update it, it does help when we actually print the diff.